Specs/Number/Japanese/PercentModel.json

```diff
@@ -15,7 +15,7 @@
 ,
 {
   "Input": "この車は19パーセントの確率が遅れた",
-  "NotSupported": "javascript, python",
+  "NotSupported": "dotNet, javascript, python",
```

Comment:
Please also add "java" to the list of NotSupported platforms.
I recommend adding NotSupported for dotnet only, as this means "currently not supported, but implementation ongoing"
While the others should be NotSupportedByDesign, a separate tag that means "an implementation in these platforms doesn't exist yet and the effort for that has not started".

Reply:
oh I see.. I modified the json an hour ago.